policy.md

```diff
@@ -226,24 +317,205 @@ Each audit statement MUST be accompanied by documentation of the audit team qual
+| If the subject hierarchy is dedicated to… | EKU Presence? | EKU OID Permitted? |
+|---|---|---|
+| TLS server authentication | MUST | Only 1.3.6.1.5.5.7.3.1                       |
+| TLS client authentication | MUST | Only 1.3.6.1.5.5.7.3.2                       |
+| TLS (generic)             | MUST | Only 1.3.6.1.5.5.7.3.1 and 1.3.6.1.5.5.7.3.2 |
+| Code signing              | MUST | Only 1.3.6.1.5.5.7.3.3                       |
+| S/MIME                    | MUST | Only 1.3.6.1.5.5.7.3.4                       |
 
-CA Owners SHOULD always use the latest available version of the self-assessment template. CA Owners MUST NOT use a version of the self-assessment template that has been superseded by more than 90 calendar days before their submission.
+In cases where both the Issuer and the Subject hierarchies are capable of issuing Extended Validation (EV) certificates, the cross-certificate MUST include the EV CA/Browser Forum Reserved Certificate Policy Identifier of 2.23.140.1.1 as a policyIdentifier. Additional policyIdentifiers MAY be present.
 
-## 6. Mailshots ##
+### 6.4 Annual CCADB Self-Assessments
 
-From time to time, a Store MAY use the CCADB to send information to CA Owners. Mailshots will be sent to at least the Primary POC(s) and the email aliases, and may also go to one or more of the other POCs; the exact recipient list is defined by the Store sending the information.
+At least annually, CA Owners MUST complete a self-assessment using [this](https://www.ccadb.org/cas/self-assessment) template to attest to the conformance of the CA Owner’s policies against industry policies and practices. CA Owners SHOULD always use the latest available version of the self-assessment template. CA Owners MUST NOT use a version of the self-assessment template that has been superseded by more than 90 calendar days before their submission.
+
+A single self-assessment MAY cover multiple CAs operating under both the same CP and CPS(s), or combined CP/CPS. CAs not operated under the same CP and CPS(s) or combined CP/CPS MUST be covered in a separate self-assessment.
+
+The self-assessment submission date is determined by the earliest appearing "BR Audit Period End Date" field specified in any of the CA Owner’s "CA Owner/Certificate" CCADB root records that are included in one or more Root Stores. 
+
+An annual self-assessment MUST be completed and submitted to the CCADB within 92 calendar days from the CA Owner's earliest appearing root record "BR Audit Period End Date." CA Owners SHOULD submit the self-assessment to the CCADB at the same time as uploading audit reports.
+
+## 7. Mailshots
+
+From time to time, a Root Store Operator MAY use the CCADB to send information to CA Owners. Mailshots will be sent to at least the Primary POC(s) and the email aliases, and may also go to one or more of the other POCs; the exact recipient list is defined by the Root Store Operator sending the information.
```

Comment:
Revise "and the email aliases" to "and the CA Owners' email aliases".

Reply:
Addressed in https://github.com/ryancdickson/www.ccadb.org/commit/2ff22b3e418c7d484b802497b67cc06bea7c8202.